pyccel/ast/sympy_helper.py

```diff
@@ -139,7 +142,7 @@ def pyccel_to_sympy(expr, symbol_map, used_names):
 
     elif isinstance(expr, PyccelMul):
         args = [pyccel_to_sympy(e, symbol_map, used_names) for e in expr.args]
-        return sp.Mul(*args)
+        return args[0] * args[1]
```

Comment:
Is our `PyccelMul` a binary operation, or can it have more than two arguments? Sorry but I don't remember

Reply:
Yes, `PyccelMul` inherits from `PyccelBinaryOperator`. This mimicks the way python's ast handles this (arithmetic operators are all instances of `ast.BinOp`)

Reply:
Thanks